lucene/core/src/test/org/apache/lucene/codecs/lucene60/TestLucene60PointsFormat.java

```diff
@@ -132,7 +132,7 @@ public void testEstimatePointCount() throws IOException {
     PointValues points = lr.getPointValues("f");
 
     // If all points match, then the point count is numLeaves * maxPointsInLeafNode
-    final int numLeaves = (int) Math.ceil((double) points.size() / maxPointsInLeafNode);
+    final int numLeaves = (int) Math.max(Math.ceil((double) points.size() / maxPointsInLeafNode), 1);
```

Comment:
I don't think this line needs to change, we already get a number of leaves equal to 1 when there are fewer points than number of points per leaf?

Reply:
Yes, you are right, I reverted that change.